### PR TITLE
BSD fixes #232: Use only uswds core and usa skipnav instead of ALL of all uswds components.

### LINKED ISSUE #232: Remove unused CSS and Drupal libraries
We're currently loading the entire USWDS CSS file, but we're not using many of the components.

Screenshot of entire USWDS CSS file being loaded:
image

## Proposed solution

Identify USWDS components we really need and use SCSS packages to **only** load those styles.

## Additional context

Semi-related to #125.

### PR DESCRIPTION
**Pull request by mattsqd**
Super tiny now
image

https://pr-233-imhdbhq-tsvj5tw7p3f66.us.platformsh.site/

### CODE REVIEW

File: web/themes/custom/bixal_uswds/src/sass/styles.scss [hunk 1]
```diff
@@ -5,7 +5,8 @@
 
 // USWDS individual components
 // to include everything, just code it as @forward "uswds";
-@forward "uswds";
+@forward "uswds-core";
+@forward "usa-skipnav";
```
Comment (mejiaj) [inline web/themes/custom/bixal_uswds/src/sass/styles.scss:9]: Skipnav isn't displaying on focus, but we can target that in another issue if its a blocker. Seems to be also happening on production.

| Expected | Actual |
| :----- | :---- |
| | |

**Comment by mattsqd** [comment]
@mejiaj  I can't promise it's amazing, but I think the skip nav link is shown when focused now!
https://pr-233-imhdbhq-tsvj5tw7p3f66.us.platformsh.site/

**Comment by buddy-harlow** [comment]
@mattsqd I still can't see the skip nav link, but I do see it in the source.

**Comment by mattsqd** [comment]
> @mattsqd I still can't see the skip nav link, but I do see it in the source.

The easiest way to get it to show is to reload the page, then immediately hit tab. Should look like this:
image

**Comment by buddy-harlow** [comment]
Gotcha! Strangely, if you try to look at it with inspect open it's like hidden at the top and just barely visible.

**Review by buddy-harlow — APPROVED** [review]

**Pull request merged**